src/ray/protobuf/node_manager.proto

```diff
@@ -169,7 +169,9 @@ message PinObjectIDsRequest {
   repeated bytes object_ids = 2;
 }
 
-message PinObjectIDsReply {}
+message PinObjectIDsReply {
+  repeated bool success = 1;
```

Comment:
nit: should we use plural here?

Reply:
Addressed in https://github.com/ray-project/ray/pull/25104 since I split the PR into two.